engines/sci/engine/script_patches.cpp

```diff
@@ -7146,22 +7148,22 @@ static const uint16 qfg1vgaSignatureWhiteStagDagger[] = {
 	0x1a,                               // eq?
 	0x31, 0x16,                         // bnt [next parameter check]
 	0x76,                               // push0
-	0x45, 0x02, 0x00,                   // callb export 2 from script 0, 0
```

Comment:
Maybe this notation for CALLBs may even be better, because it makes it obvious what the call actually does.
"call export 2 from script 0". Anyone should understand that.